tests/system_tests/test_blueapi_system.py

```diff
@@ -178,14 +264,7 @@ def test_set_state_transition_error(client: BlueapiClient):
 def test_get_task_by_status(client: BlueapiClient):
     task_1 = client.create_task(_SIMPLE_TASK)
     task_2 = client.create_task(_SIMPLE_TASK)
-    task_by_pending_request = requests.get(
-        client._rest._url("/tasks"), params={"task_status": TaskStatusEnum.PENDING}
-    )
-    assert task_by_pending_request.status_code == status.HTTP_200_OK
-    task_by_pending = TypeAdapter(TasksListResponse).validate_python(
-        task_by_pending_request.json()
-    )
-
+    task_by_pending = client.get_all_tasks()
```

Comment:
This is now getting all tasks, not just those that are pending, which is a change of the expected behaviour